syncr_backend/util/drop_util.py

```diff
@@ -407,6 +409,47 @@ class PermissionError(Exception):
     return metadata
 
 
+async def check_for_changes(drop_id: bytes) -> Optional[Set[str]]:
```

Comment:
This is good for now, but in the future it might make sense to have this return (unchanged, changed, added, deleted), so the frontend can easily show that.

Reply:
Should it be a list of tuples with every file that is in both with a flag depending on its status?

Reply:
I think each file should be in exactly one of `unmodified, modified, added, deleted`?  And then another pass could be used to detect moved files (aka a file with the same hash in both added and deleted).

Reply:
Ill do that

Reply:
Probably used something like [namedtuple](https://docs.python.org/3.6/library/collections.html#collections.namedtuple) ([mypy docs](https://mypy.readthedocs.io/en/latest/kinds_of_types.html#named-tuples)) instead of just a tuple to avoid confusion.